lib/public_key/src/pubkey_cert.erl

```diff
@@ -370,6 +370,9 @@ select_extension(_, asn1_NOVALUE) ->
     undefined;
 select_extension(_, []) ->
     undefined;
+select_extension(Id, [#'Extension'{extnID = ?'id-ce-cRLDistributionPoints' = Id,
+                                   extnValue = Value} = Extension | _]) when is_binary(Value) ->
+    Extension#'Extension'{extnValue = public_key:der_decode('CRLDistributionPoints', Value)};
 select_extension(Id, [#'Extension'{extnID = Id} = Extension | _]) ->
     Extension;
```

Comment:
Why is it only the extension ?id-ce-cRLDistributionPoints that is decoded here? Could there not be other extensions as well that is not decoded until used?
Would a test of `is_binary(Value)` imply that the extension needs to be decoded here?

Reply:
This CRL extension is not decode initially  but then decode later when used by public_key functions.  Other  possibly unencoded  extensions are not know by public key and handled totally by some other code that will have to do the decoding. We could possible find a reason to add more known public_key extensions here later, but I do not think there is an obvious one at the moment.  Yes is_binary implies that  the value is DER encoded.